tests/bootstrap.php

```diff
@@ -3,17 +3,21 @@
 date_default_timezone_set('UTC');
 
 // Include the composer autoloader
-$loader = require __DIR__ . '/../vendor/autoload.php';
+$loader = require __DIR__.'/../vendor/autoload.php';
 
-if (!class_exists('\PHPUnit\Framework\Constraint\RegularExpression')) {
+if ( ! defined('PHPUNIT_COMPOSER_INSTALL')) {
```

Comment:
Can you change the not operator spacing in this file as well? i.e. (!class_exists...